### ISSUE TITLE
Wrong translation of grid element

### ISSUE THREAD

**Issue by tsh-gmbh** [laxap/bootstrap_grids#22]
Hello,

if i translate a grid element within the containing elements (it appears on all CTypes), the english pendant is being created and additionaly the german content will be copied to the new grid in the target language.

So you have to delete the content with the attribute: "Value is not allowed "0"" in the language field of the target CType.

My config:
bootstrap_package
bootstrap_grids
TYPO3 7.6.12

All extensions are installed in the last version.

![bug](https://cloud.githubusercontent.com/assets/21078739/20019310/fc0d9442-a2cb-11e6-84d5-9b448aa75292.JPG)

**Comment by laxap** [comment]
Hello

I don't think it is related to bootstrap_package but I am not sure since I did not develop the extension. I tested bootstrap_grids only with the distribution extension "bsdist" and without any bootstrap extension.

I just tested the translation of grids and it works fine. But when I translate the elements the language of the translated elements is set to 1, not 0. Zero is used for the default language.

Regards, Pascal
